editor/store/actions.js

```diff
@@ -337,6 +337,12 @@ export function savePost() {
 	};
 }
 
+export function refreshPost() {
+	return {
+		type: 'UPDATE_POST_FROM_SERVER',
```

Comment:
Any particular reason the `type` and function name don't align? Why not call the function `updatePostFromServer`, or the `type` `REFRESH_POST` ?

Reply:
Just a brain fart. 🙃